indico_themes_canonical/static/css/ubuntu-summit/header.css

```diff
@@ -47,22 +35,62 @@ Wraps around the title box
 */
 .confTitleBox {
   background: none;
+  display: flex;
+  margin: auto;
+  padding: 80px 20px;
   text-align: left;
-}
-
-.confTitle h1 {
-  width: auto;
+  width: var(--full-width);
 }
 
 /*
 The style for the title text.
 */
-.conference-title-link {
-  font-size: 2.228rem;
+.confTitle {
+  display: inline-block;
+  margin-left: var(--global-margin-left);
+  width: 100%;
+}
+
+.headerLogoCont {
```

Comment:
Same here, could this not be `header-logo-cont`?

Reply:
Same here, previous classes